### ISSUE TITLE
improve message when failing to get external congif

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABServer#5742]
when Crab Server fails to retrieve external config from GITLAB it can fail to start with
somehow cryptic messages.
See the thread  below
```


Maybe one easy thing to do is to improve the error message here
https://github.com/dmwm/CRABServer/blob/master/src/python/CRABInterface/Utils.py#L115

We should not assume the next person that will face the error in 4
years from now will know what is going on. Something like "Internal
issue when retrieving external confifuration at %s. If the link is not
available please wait that it comes back online and try to start the
crabserver again." % extconfigurl
On Thu, Sep 27, 2018 at 2:33 PM Alan Malta Rodrigues <alan.malta@cern.ch> wrote:
>
> Agreed! As I always say, nothing is just 
>
> Cheers,
> Alan.
> ________________________________________
> From: Stefano Belforte
> Sent: Thursday, September 27, 2018 2:30 PM
> To: Alan Malta Rodrigues; Marco Mascheroni
> Cc: Stefano Belforte; Lina Marcela Jimenez Becerra; cms-service-webtools (Announcements from IT and VOC to Webtools. Discussions am...)
> Subject: Re: crabserver can't be restarted(?)
>
> perfectly true Alan. I think we average O(20)
> changes a year (e.g. use that file to remove sick schedd
> from the rotation).  But since even small changes are
> not that cheap in a project with no developer, I
> will put the little effort I can give to CRAB in the
> areas where we have real problems, like publication atm.
> stefano
>
> On 27/09/18 14:25, Alan Malta Rodrigues wrote:
>> If that json changes quite often, then failing to start the service isn't
>> that bad indeed. However, if it changes only a couple of times per
>> year, the effort to persist it on disk (which I believe to be small) would
>> probably payoff.
>>
>> Anyways, you both know crabserver much better than me.
>> Cheers,
>> Alan.
>> ________________________________________
>> From: Stefano Belforte
>> Sent: Thursday, September 27, 2018 11:49 AM
>> To: Marco Mascheroni
>> Cc: Stefano Belforte; Lina Marcela Jimenez Becerra; Alan Malta Rodrigues; cms-service-webtools (Announcements from IT and VOC to Webtools. Discussions am...)
>> Subject: Re: crabserver can't be restarted(?)
>>
>> thanks for clarifying. I thought cache was on disk.
>> I still agree to live with it. Actually such a failure at
>> server start time is healthy, IMO.
>> Stefano
>>
>> On 27/09/18 11:45, Marco Mascheroni wrote:
>>> the cache is in memory, and can't do much if git is down when we
>>> start the crabserver.. I think we went through this already many years
>>> ago and decided that this event occurring (gitlab down and we starting
>>> the crabserver) is so rare that we can live with it.
>>

```

**Comment by belforte** [comment]
addressed in https://github.com/dmwm/CRABServer/pull/5741
to be tested

**Comment by belforte** [comment]
merged in master and tagged in 3.3.1810.patch5
